From 9fe4c3c5cd0dbbec327b9c6cd5f56eabf4ed59c4 Mon Sep 17 00:00:00 2001
From: Tristan de Cacqueray <tristanC@wombatt.eu>
Date: Sun, 14 Apr 2019 02:50:09 +0000
Subject: [PATCH] license: update fsf address

License headers are using the old FSF address and this is causing
packaging error with rpmlint: incorrect-fsf-address.
---
 nova-tt/dummy_mutex.hpp              | 4 ++--
 nova-tt/mlock.hpp                    | 4 ++--
 nova-tt/name_thread.hpp              | 4 ++--
 nova-tt/nanosleep.hpp                | 4 ++--
 nova-tt/pause.hpp                    | 4 ++--
 nova-tt/physical_concurrency.hpp     | 4 ++--
 nova-tt/rw_mutex.hpp                 | 4 ++--
 nova-tt/rw_spinlock.hpp              | 4 ++--
 nova-tt/semaphore.hpp                | 4 ++--
 nova-tt/semaphore_boost_fallback.hpp | 4 ++--
 nova-tt/semaphore_dispatch.hpp       | 4 ++--
 nova-tt/semaphore_mach.hpp           | 4 ++--
 nova-tt/semaphore_posix.hpp          | 4 ++--
 nova-tt/semaphore_pthreads.hpp       | 4 ++--
 nova-tt/semaphore_win32.hpp          | 4 ++--
 nova-tt/spin_lock.hpp                | 4 ++--
 nova-tt/thread_affinity.hpp          | 4 ++--
 nova-tt/thread_pool.hpp              | 4 ++--
 nova-tt/thread_priority.hpp          | 4 ++--
 nova-tt/thread_priority_fallback.hpp | 4 ++--
 nova-tt/thread_priority_mach.hpp     | 4 ++--
 nova-tt/thread_priority_pthread.hpp  | 4 ++--
 nova-tt/thread_priority_win32.hpp    | 4 ++--
 23 files changed, 46 insertions(+), 46 deletions(-)

diff --git a/nova-tt/dummy_mutex.hpp b/nova-tt/dummy_mutex.hpp
index ed9915f..7f6f6f6 100644
--- a/nova-tt/dummy_mutex.hpp
+++ b/nova-tt/dummy_mutex.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file dummy_mutex.hpp */
 
diff --git a/nova-tt/mlock.hpp b/nova-tt/mlock.hpp
index 95f6666..f66b944 100644
--- a/nova-tt/mlock.hpp
+++ b/nova-tt/mlock.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file mlock.hpp */
 
diff --git a/nova-tt/name_thread.hpp b/nova-tt/name_thread.hpp
index afeda17..4cc6efe 100644
--- a/nova-tt/name_thread.hpp
+++ b/nova-tt/name_thread.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file mlock.hpp */
 
diff --git a/nova-tt/nanosleep.hpp b/nova-tt/nanosleep.hpp
index 5b387a7..f0c0cb2 100644
--- a/nova-tt/nanosleep.hpp
+++ b/nova-tt/nanosleep.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file nanosleep.hpp */
 /** \namespace nova */
diff --git a/nova-tt/pause.hpp b/nova-tt/pause.hpp
index 27b0b30..035be69 100644
--- a/nova-tt/pause.hpp
+++ b/nova-tt/pause.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file spin_lock.hpp */
 
diff --git a/nova-tt/physical_concurrency.hpp b/nova-tt/physical_concurrency.hpp
index 8b216a3..2c4093a 100644
--- a/nova-tt/physical_concurrency.hpp
+++ b/nova-tt/physical_concurrency.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_PHYSICAL_CONCURRENCT_HPP
 #define NOVA_TT_PHYSICAL_CONCURRENCT_HPP
diff --git a/nova-tt/rw_mutex.hpp b/nova-tt/rw_mutex.hpp
index e254381..14dfa4c 100644
--- a/nova-tt/rw_mutex.hpp
+++ b/nova-tt/rw_mutex.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file rw_mutex.hpp */
 
diff --git a/nova-tt/rw_spinlock.hpp b/nova-tt/rw_spinlock.hpp
index d106363..a33b047 100644
--- a/nova-tt/rw_spinlock.hpp
+++ b/nova-tt/rw_spinlock.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file rw_spinlock.hpp */
 
diff --git a/nova-tt/semaphore.hpp b/nova-tt/semaphore.hpp
index 2035952..12db2e2 100644
--- a/nova-tt/semaphore.hpp
+++ b/nova-tt/semaphore.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file semaphore.hpp */
 
diff --git a/nova-tt/semaphore_boost_fallback.hpp b/nova-tt/semaphore_boost_fallback.hpp
index fe825ee..1970190 100644
--- a/nova-tt/semaphore_boost_fallback.hpp
+++ b/nova-tt/semaphore_boost_fallback.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_BOOST_FALLBACK_HPP
 #define NOVA_TT_SEMAPHORE_BOOST_FALLBACK_HPP
diff --git a/nova-tt/semaphore_dispatch.hpp b/nova-tt/semaphore_dispatch.hpp
index 1cee7a2..0832666 100644
--- a/nova-tt/semaphore_dispatch.hpp
+++ b/nova-tt/semaphore_dispatch.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_DISPATCH_HPP
 #define NOVA_TT_SEMAPHORE_DISPATCH_HPP
diff --git a/nova-tt/semaphore_mach.hpp b/nova-tt/semaphore_mach.hpp
index 4e9ed46..c89f045 100644
--- a/nova-tt/semaphore_mach.hpp
+++ b/nova-tt/semaphore_mach.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_MACH_HPP
 #define NOVA_TT_SEMAPHORE_MACH_HPP
diff --git a/nova-tt/semaphore_posix.hpp b/nova-tt/semaphore_posix.hpp
index 0c9b900..64f5ff1 100644
--- a/nova-tt/semaphore_posix.hpp
+++ b/nova-tt/semaphore_posix.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_POSIX_HPP
 #define NOVA_TT_SEMAPHORE_POSIX_HPP
diff --git a/nova-tt/semaphore_pthreads.hpp b/nova-tt/semaphore_pthreads.hpp
index 7c02b9b..477b674 100644
--- a/nova-tt/semaphore_pthreads.hpp
+++ b/nova-tt/semaphore_pthreads.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_PTHREADS_HPP
 #define NOVA_TT_SEMAPHORE_PTHREADS_HPP
diff --git a/nova-tt/semaphore_win32.hpp b/nova-tt/semaphore_win32.hpp
index 8741b6c..4877821 100644
--- a/nova-tt/semaphore_win32.hpp
+++ b/nova-tt/semaphore_win32.hpp
@@ -15,8 +15,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_SEMAPHORE_WIN32_HPP
 #define NOVA_TT_SEMAPHORE_WIN32_HPP
diff --git a/nova-tt/spin_lock.hpp b/nova-tt/spin_lock.hpp
index 76a91c6..abfbcc3 100644
--- a/nova-tt/spin_lock.hpp
+++ b/nova-tt/spin_lock.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file spin_lock.hpp */
 
diff --git a/nova-tt/thread_affinity.hpp b/nova-tt/thread_affinity.hpp
index 2abe4e8..e0bb98f 100644
--- a/nova-tt/thread_affinity.hpp
+++ b/nova-tt/thread_affinity.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 /** \file thread_affinity.hpp */
 
diff --git a/nova-tt/thread_pool.hpp b/nova-tt/thread_pool.hpp
index adc5b33..60078b7 100644
--- a/nova-tt/thread_pool.hpp
+++ b/nova-tt/thread_pool.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef THREAD_POOL_HPP
 #define THREAD_POOL_HPP
diff --git a/nova-tt/thread_priority.hpp b/nova-tt/thread_priority.hpp
index 1d3f0a7..ede613b 100644
--- a/nova-tt/thread_priority.hpp
+++ b/nova-tt/thread_priority.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_THREAD_PRITORITY_HPP
 #define NOVA_TT_THREAD_PRITORITY_HPP
diff --git a/nova-tt/thread_priority_fallback.hpp b/nova-tt/thread_priority_fallback.hpp
index 8548ecd..4386cfd 100644
--- a/nova-tt/thread_priority_fallback.hpp
+++ b/nova-tt/thread_priority_fallback.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_THREAD_PRITORITY_FALLBACK_HPP
 #define NOVA_TT_THREAD_PRITORITY_FALLBACK_HPP
diff --git a/nova-tt/thread_priority_mach.hpp b/nova-tt/thread_priority_mach.hpp
index 6b91c56..0318ebe 100644
--- a/nova-tt/thread_priority_mach.hpp
+++ b/nova-tt/thread_priority_mach.hpp
@@ -14,8 +14,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_THREAD_PRITORITY_MACH_HPP
 #define NOVA_TT_THREAD_PRITORITY_MACH_HPP
diff --git a/nova-tt/thread_priority_pthread.hpp b/nova-tt/thread_priority_pthread.hpp
index 18f8b12..b4fb18b 100644
--- a/nova-tt/thread_priority_pthread.hpp
+++ b/nova-tt/thread_priority_pthread.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_THREAD_PRITORITY_PTHREAD_HPP
 #define NOVA_TT_THREAD_PRITORITY_PTHREAD_HPP
diff --git a/nova-tt/thread_priority_win32.hpp b/nova-tt/thread_priority_win32.hpp
index 1caab83..31c3c2e 100644
--- a/nova-tt/thread_priority_win32.hpp
+++ b/nova-tt/thread_priority_win32.hpp
@@ -13,8 +13,8 @@
 //
 //  You should have received a copy of the GNU General Public License
 //  along with this program; see the file COPYING.  If not, write to
-//  the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-//  Boston, MA 02111-1307, USA.
+//  the Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
+//  Boston, MA 02110-1301, USA.
 
 #ifndef NOVA_TT_THREAD_PRITORITY_WIN32_HPP
 #define NOVA_TT_THREAD_PRITORITY_WIN32_HPP
-- 
2.21.0

